src/popup/router/components/NameItem.vue

```diff
@@ -317,11 +317,17 @@ export default {
     .input-field {
       margin: 8px 0;
 
-      ::v-deep .wrapper {
+      ::v-deep .input-wrapper {
```

Comment:
In general styling child component's elements is a bad thing and we should do it.

Reply:
I did not introduce it in that PR, and i don't think we have time for now to change the approach.